### ISSUE TITLE
`packet capacity get` output is hard to grep

### ISSUE THREAD

**Issue by vielmetti** [equinix/metal-cli#80]
### What happened?

I'm trying to figure out capacity of a particular machine type (c2.large.arm)
across all data centers. It seems like I should be able to answer this query
with something like

```
packet capacity get --plan c2.large.arm
```

However, capacity doesn't take --plan. so the next step is to just do

```
packet capacity get | grep c2.large.arm
```

except that by inspection capacity draws funny boxes around the data,
and omits duplicate values in a column, making it unsuitable for grep:

```
emv@pinateck:~/src/github.com/packethost/packet-cli/bin $ ./packet capacity get | grep c2.large.arm 
|          | c2.large.arm  |             |
| nrt1     | c2.large.arm  | limited     |
|          | c2.large.arm  | limited     |
|          | c2.large.arm  | unavailable |
|          | c2.large.arm  | unavailable |
```

### How can we reproduce it?

CLI version (`packet --version`): packet version 9ce0ac0

**Comment by vielmetti** [comment]
The underlying code that renders tables is pretty functional; https://github.com/olekukonko/tablewriter is the source for it.

**Comment by vielmetti** [comment]
This diff fixes it for my purposes. PR to follow.

```
diff --git a/cmd/retrieve_capacity.go b/cmd/retrieve_capacity.go
index 6640d02..e41818d 100644
--- a/cmd/retrieve_capacity.go
+++ b/cmd/retrieve_capacity.go
@@ -51,7 +51,7 @@ packet capacity get
                        }
                }
 
-               return outputMergingCells(capacities, header, &requiredDataFormat)
+               return output(capacities, header, &requiredDataFormat)
        },
 }
```

**Comment by displague** [comment]
While refactoring this project in #133, I noticed that this grouped table formatting was only applied for this one command. I agree that the result is hard to parse and the output is visually confusing and distracting. The grouping algorithm makes subsequent runs (even moments apart) sort in a completely different order, with different grouping boxes chosen.

I'm going to remove this odd formatting as part of #128.

In the meanwhile, `-y` or `-j` provide YAML and JSON outputs which are easier to evaluate, especially with `jq`.

**Comment by displague** [comment]
The merged cells are no longer returned.
